config/database.yml.example

```diff
@@ -2,15 +2,15 @@ development:
   adapter: postgresql
   encoding: unicode
   database: simple_todo_app_development
-  # username: your_username
```

Comment:
You should leave `config/database.yml.example` unmodified and only put your real credentials in `config/database.yml`